mlflow/models/docker_utils.py

```diff
@@ -80,10 +82,10 @@ def _get_maven_proxy():
 
 _DOCKERFILE_TEMPLATE = """
 # Build an image that can serve mlflow models.
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
 RUN apt-get -y update
-RUN apt-get install -y --no-install-recommends \
+RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y --no-install-recommends \
```

Comment:
Build hangs due to tzdata's prompt. `DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC` prevents it.